README.md

```diff
@@ -25,55 +26,27 @@ The Busola also includes React and Angular libraries:
 - [`npm`](https://www.npmjs.com/): >= 6.4.0
 - [`node`](https://nodejs.org/en/): >= 12.0.0
 
-## Installation
-
-1. Install [Kyma](https://kyma-project.io/docs/master/root/kyma/#installation-install-kyma-locally) as a backing service for your local instance of Busola. Make sure you import certificates into your operating system and mark them as trusted. Otherwise, you cannot access the applications hosted in the `kyma.local` domain.
-
-2. Install Busola dependencies. To install dependencies for the root and all UI projects, and prepare symlinks for local libraries within this repository, run the following command:
+## Usage with NPX
 
-   ```bash
-   npm run bootstrap
-   ```
+This mode has no hot-reload feature so it's not designed for the development. Hovewer, it gives you the ability to run the Busola in a few seconds without installing any dependencies.
 
-   > **NOTE:** The `npm run bootstrap` command:
-   >
-   > - installs root dependencies provided in the [package.json](./package.json) file
-   > - installs dependencies for the [`React common`](./common), [`React components`](./components/react), [`Shared components`](./components/shared) and [`Generic documentation`](./components/generic-documentation) libraries
-   > - builds all the libraries
-   > - installs dependencies for all the [components](#components)
-   > - updates your `/etc/hosts` with the `127.0.0.1 busola-dev.kyma.local` host
-   > - creates the `.clusterConfig.gen` file if it doesn't exist, pointing at the `kyma.local` domain
+To run the Busola in the `npx` mode from the latest relase type `npx @kyma-project/busola@latest` in your terminal. That's it, your default browser should open the Busola within a few seconds.
```

Comment:
Maybe "from the latest release ~type~:.

And moreover, you need to have a kubeconfig exported in your env.

Reply:
```
type `npx @kyma-project/busola@latest` in your terminal
```
I think it's correct...


Regarding the kubeconfig, I'll include this info in the readme 👍